apps/system/test/apps/wifi-direct/js/main.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+(function() {
+  var list = document.querySelector('section ul');
```

Comment:
I would suggest to add a class name to the ul element and use it in the selector.